src/ripple/ripple.js

```diff
@@ -118,7 +119,12 @@ MaterialRipple.prototype.upHandler_ = function(event) {
 
   // Don't fire for the artificial "mouseup" generated by a double-click.
   if (event && event.detail !== 2) {
-    this.rippleElement_.classList.remove(this.CssClasses_.IS_VISIBLE);
+    // Allow a repaint to occur before removing this class, so the animation
+    // shows for tap events, which seem to trigger a mouseup too soon after
+    // mousedown.
+    window.setTimeout(function() {
```

Comment:
I would perhaps in a pedantic world :fire: :moon:  argue for RAF or a setTimeout of 17 ... but this is not a big deal at all :) It was a fun learning experience :books: